src/app/renderer/components/app/FileBrowser/components/FileEditor/components/Entry/Entry.tsx

```diff
@@ -24,7 +24,7 @@ export class Entry extends React.Component<Props, {}> {
     render() {
         return (
             <EntryWrapper
-                headline={this.props.entry.id as string || ''}
+                headline={this.props.entry.id as string || '0'}
```

Comment:
this is wrong. it should display an empty string when id is not there. Now you need to allow displaying 0 as well